Add AKS onboarding form

malcolmbaig opened the pull request:
### Context

We're migrating AYTQ/CTR to AKS. 
<!-- Why are you making this change? -->

### Changes proposed in this pull request
Kick off the process by completing this onboarding form ([source](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-form-template.md)) to share with [teacher_services_infra](https://ukgovernmentdfe.slack.com/archives/C011EM7HU85).
<!-- Include a summary of the change. -->
<!-- Why this particular solution? -->
<!-- What assumptions have you made? -->
<!-- Are there any side effects to note? -->
<!-- If there are UI changes, please include Before and After screenshots. -->

### Guidance to review

<!-- How could someone else check this work? -->
<!-- Which parts do you want more feedback on? -->

### Link to Trello card
https://trello.com/c/Ng21PrA6/130-start-aytq-aks-migration-by-completing-onboarding-form-and-generating-initial-code
<!-- http://trello.com/123-example-card -->

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

malcolmbaig committed:
Add AKS onboarding form

malcolmbaig commented on docs/aks/onboarding-form.md:
```diff
@@ -0,0 +1,45 @@
+  - access-your-teaching-qualifications.teacherservices.cloud
+  - check-a-teachers-record.teacherservices.cloud
+  - test._ and preprod._ variants of the above.
+- Service short name? aytqctr
```
The existing service short name is just aytq — using aytqctr here so it's a little more obvious that this app consists of two services.

saliceti replied:
I think it's limited to 6 characters. Maybe atqctr? or tqtr? or tqr?
Or keep aytq as it's the name of the Azure Product tag and there can be only 1.

malcolmbaig replied:
Let's keep aytq 👍🏼

malcolmbaig commented on docs/aks/onboarding-form.md:
```diff
@@ -0,0 +1,45 @@
+- Environments? e.g. develoment, staging, production
+  - test
+  - preprod
+  - production
```
@felixclack I've opted not to have a dev environment here as it's always seemed a little redundant. Children's barred list took the same approach when it was set up on AKS.

felixclack replied:
Makes sense to me. I never used the deployed dev environment.

saliceti commented on docs/aks/onboarding-form.md:
```diff
@@ -0,0 +1,45 @@
+- List of developers who will need access to Azure? Do they use a DfE laptop or BYOD? e.g
+  - Felix Clack - BYOD
+  - James Gunn - BYOD
+  - Richard Pattinson - DfE laptop
```
@malcolmbaig you are owner of the `s189 TRA delivery team` group. You can add Richard as member

saliceti commented on docs/aks/onboarding-form.md:
```diff
@@ -0,0 +1,45 @@
+- Github repos / Docker registry type: Open (public) by default, or indicate any requirement to make them private
+  - Open
+- Namespaces e.g. bat-qa (test cluster), bat-staging (test cluster), bat-production (production cluster)
+  - n/a
```
You will most certainly use the tra namespaces. Today there are: tra-development, tra-test, tra-production. You can decide how to map the environments to the namespaces.

malcolmbaig committed:
Amend AKS onboarding form with PR feedback

- Use aytq as short name, due to 6 char limit
- Add in tra- namespaces with mapping to environments
